README.md

```diff
@@ -15,7 +15,7 @@ and only communicate with state changes in "object values" (bindings), not widge
 
 ### Examples
 
-- [Calculator](https://github.com/MasterQ32/Dunstwolke/tree/master/dunstwolke/examples/calculactor)
+- [Calculator](https://github.com/Avokadoen/core/tree/master/src/examples)
```

Comment:
This should be a relative link, and neither a link to my or your github account

Reply:
Ooof, I guess I need more coffee. Sorry about that